package.json

```diff
@@ -26,7 +26,9 @@
   },
   "homepage": "https://github.com/Microsoft/vscode-java-debug/blob/master/README.md",
   "categories": [
-    "Debuggers"
+    "Debuggers",
```

Comment:
Could add categories `Java`?

Reply:
AFAIK, the value is a enum. No such category called Java.